src/test/ui/break-outside-loop-val.rs

```diff
@@ -0,0 +1,7 @@
+// Breaks with values inside closures used to ICE (#66863)
```

Comment:
Let's move this to `ui/issues/issue-66863-break-outside-loop-val` to avoid cluttering `ui/`

Reply:
I think you meant `issue-66702` ?